atlasdb-client-protobufs/build.gradle

```diff
@@ -1,15 +1,6 @@
 apply from: "../gradle/publish-jars.gradle"
 apply from: "../gradle/shared.gradle"
```

Comment:
this package (and the leader-election-api-protobuf) are a bit odd in general considering they only exist to contain generated code.  I'm wondering if we should just throw it all into there source.  I've reached out to @gbonik hoping to find out more about who actually consumes these.